docs/source/scenarios/container_inventory_management.rst

```diff
@@ -633,11 +633,86 @@ random mode, we hope this could help you learn how to use the environment interf
 
            # Drive the environment with the random action.
            metrics, decision_event, is_done = env.step(action)
-
+    
        # Query for the environment business metrics at the end of each episode,
        # it is usually users' optimized object in CIM scenario (usually includes multi-target).
        print(f"ep: {ep}, environment metrics: {env.metrics}")
        env.reset()
 
 Jump to `this notebook <https://github.com/microsoft/maro/tree/master/notebooks/container_inventory_management/interact_with_environment.ipynb>`_
 for a quick experience.
+
+
+Inter-view
+~~~~~~~~~~
+
+Generally speaking, the number of epochs in this scene will be greater
```

Comment:
scenario instead of scene

Reply:
BTW, I think the description is not right. For rule-based or OR-based strategies, maybe only 1 epoch is enough.